buildbot-conf/cordova-config.json.sample

```diff
@@ -0,0 +1,43 @@
+{
+    "couchdb": {
+        "uri":  "http://localhost:5984"
+    },
+    "dashboard": {
+        "port": 8088
+    },
+    "app": {
+        "entry":   "autotest/pages/all.html",
+        "release": "master",
```

Comment:
I was hesitating to remove it, because I was unsure if it was unused. Removed now.